### PR TITLE
Added Download attachment action to Plumsail Forms connector

### PR DESCRIPTION
**Pull request by jusper-dk**
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

File: certified-connectors/Plumsail Forms/apiDefinition.swagger.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-    "version": "v1",
     "title": "Plumsail Forms",
     "description": "This API allows you to process Plumsail Forms submissions and their attachments.",
     "contact": {
       "name": "Plumsail Team",
       "url": "https://plumsail.com",
       "email": "support@plumsail.com"
-    }
+    },
+    "version": "v1"
```
Comment (Amjed-Ayoub) [inline certified-connectors/Plumsail Forms/apiDefinition.swagger.json:11]: @jusper-dk the latest version in ISV Studio for Plumsail Forms is version 4.0. if this is a new version, please change to "5.0".

File: certified-connectors/Plumsail Forms/apiDefinition.swagger.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "v1",
     "title": "Plumsail Forms",
     "description": "This API allows you to process Plumsail Forms submissions and their attachments.",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Plumsail Forms/apiDefinition.swagger.json:5]: @jusper-dk please remove the word "API" from the description. please follow these [requirements](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info)

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: certified-connectors/Plumsail Forms/apiDefinition.swagger.json (hunk 1 above)
Comment (jusper-dk) [reply certified-connectors/Plumsail Forms/apiDefinition.swagger.json]: Done.

File: certified-connectors/Plumsail Forms/apiDefinition.swagger.json (hunk 2 above)
Comment (jusper-dk) [reply certified-connectors/Plumsail Forms/apiDefinition.swagger.json]: Fixed.

**Review by Amjed-Ayoub — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @jusper-dk,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**